### ISSUE TITLE
Internal server error on clicking the settings icon

### ISSUE THREAD

**Issue by snassar** [hacdias/caddy-filemanager#78]
### 1. Which revision of File Manager are you running (check the footer on any page)?

a9aaba5

### 2. What are you trying to do?

I clicked the Settings icon

### 3. What is your entire Caddyfile?
```text
http://playground.dev {
    root /srv/http/playground.dev/public

    gzip

    tls off

    log /var/log/caddy/playground.dev/log {
        rotate {
            size 50
            age 30
            keep 5
        }
    }

    errors {
        log /var/log/caddy/playground.dev/errors {
            size 50
            age 30
            keep 5
        }
    }

    hugo ./ admin
}
```

### 4. What did you expect to see?

I am not sure. Maybe a config page for the site?

### 5. What did you see instead (give full error messages and/or log)?

500 Internal Server Error

Try reloading the page or hitting the back button. If this error persists, it seems that you may have found a bug! Please create an issue at hacdias/caddy-filemanager repository on GitHub with the code below.

```
template: frontmatter:2:6: executing "blocks" at <.Fields>: can't evaluate field Fields in type *frontmatter.Content
```

##
# 6. How can someone who is starting from scratch reproduce this behavior as minimally as possible?

* Install Caddy on Arch linux using https://aur.archlinux.org/packages/caddy-full-bin/
* Setup Caddyfile and config.toml

**Comment by hacdias** [comment]
Can you please give me your Hugo config file, please? So I can try to reproduce

**Comment by snassar** [comment]
Here it is:

```toml
languageCode = "en-us"
title = "Samir Nassar"
baseURL = "http://playground.dev/"
theme = "hugo-theme-bootstrap4-blog"
metaDataFormat = "toml"
```

Even with it doesn't work:

```toml
languageCode = "en-us"
title = "Samir Nassar"
baseURL = "http://playground.dev/"
theme = "hugo-theme-bootstrap4-blog"
```

**Comment by hacdias** [comment]
What's the file extension? The file name?

**Comment by hacdias** [comment]
I used the same file and I didn't get any error 😕

**Comment by hacdias** [comment]
Could you please try using the latest version, building it from source? 😄

**Comment by hacdias** [comment]
Hello @snassar! I've checked and this was already fixed in the meanwhile (between commit a9aaba5 and the most recent one). I'll close this and I hope @mholt can update this on Caddy server when he's got time
 😄
